src/packaging/docker-services/generate-ssl-stores/docker-entrypoint.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+
+set -ex
+
+CASSANDRA_KEYSTORE_PASSWORD=$1
+CASSANDRA_TRUSTSTORE_PASSWORD=$2
+
+CASSANDRA_KEYSTORE=${WORKDIR}/ssl-stores/cassandra-server-keystore.jks
+REAPER_KEYSTORE=${WORKDIR}/ssl-stores/reaper-server-keystore.jks
+GENERIC_TRUSTSTORE=${WORKDIR}/ssl-stores/generic-server-truststore.jks
+
+CA_CERT_CONFIG=${WORKDIR}/ca_cert.conf
+ROOT_CA_CERT=${WORKDIR}/ssl-stores/ca-cert
+
+echo "Generic Certificate Authority configuration"
+cat ${CA_CERT_CONFIG}
+echo
+
+rm -rf ${WORKDIR}/ssl-stores/*
```

Comment:
+1  on having that prompt and flow. Defaulting to [Y/n] on Yes to delete would be most ideal.

Reply:
Instead of clearing out the contents of the directory, I will check if any of the files can be left and overridden or if they all need to be deleted. That way we delete only what is necessary instead of the entire directory.